src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.html

```diff
@@ -1,17 +1,22 @@
-<div mat-dialog-title>
-  <div class="dialog-header">
-    <span class="dialog-handler">
-      <i class="fa fa-lock"></i>
-    </span>
-    <span class="dialog-title"> Access given to {{ data.title }}</span>
-    <span class="dialog-close" (click)="onClose()">
-      <img src="assets/images/close.svg" alt="close">
-    </span>
-  </div>
-  <div mat-dialog-close></div>
-</div>
+<p-dialog
+  *ngIf="data"
+  [visible]="visible"
+  [modal]="true"
+  [draggable]="false"
+  [closable]="false"
+>
+  <p-header>
+    <div class="dialog-header">
+      <span class="dialog-handler">
+        <i class="fa fa-lock"></i>
+      </span>
+      <span class="dialog-title"> Access given to {{ data.title }}</span>
```

Comment:
Why calling that "data"? Everything is data ;-) If it is not fetched data or anonymous data, I think that is better to name for what it is.

Reply:
fixed